lib/v2/nber/papers.js

```diff
@@ -0,0 +1,36 @@
+const got = require('@/utils/got');
+const cheerio = require('cheerio');
+const path = require('path');
+const { art } = require('@/utils/render');
+const { parseDate } = require('@/utils/parse-date');
+const { getData, parseAuthor } = require('./utils');
+
+module.exports = async (ctx) => {
+    const url = 'https://www.nber.org/api/v1/working_page_listing/contentType/working_paper/_/_/search';
+    const baseUrl = 'https://www.nber.org';
+    const data = await getData(url);
```

Comment:
Please cache the response.
Imagine someone requested both
```
/nber/news
/nber/papers
```
RSSHub will send 2 requests to the API which is the same list.